openupgradelib/openupgrade.py

```diff
@@ -2494,11 +2519,11 @@ def delete_records_safely_by_xml_id(env, xml_ids):
     """
     for xml_id in xml_ids:
         logger.debug('Deleting record for XML-ID %s', xml_id)
+        record = env.ref(xml_id, raise_if_not_found=False)
+        if not record:
+            continue
         try:
-            with env.cr.savepoint():
-                record = env.ref(xml_id, raise_if_not_found=False)
-                if record and record.exists():
-                    record.unlink()
+            safe_unlink(record, do_raise=True)
         except Exception as e:
             logger.error('Error deleting XML-ID %s: %s', xml_id, repr(e))
```

Comment:
As you are revisiting this, we can as well change this to INFO, as I found this annoying other times.

Reply:
Done